server/utils/utils.ts

```diff
@@ -162,6 +162,14 @@ export const formatName = (
   } else if (options?.style === NameFormatStyle.lastCommaFirst) {
     names.unshift(`${names.pop()},`)
     names.pop() // Remove middleNames
+  } else if (options?.style === NameFormatStyle.firstLast) {
```

Comment:
Could do with extra unit testing for this. Not sure this will handle the situation where there is no middle name(s) to start with?

Reply:
Thanks for taking a look. I'll take a closer look at that use case and write an extra unit test.

Reply:
@davidmiddletonmoj, I added a unit test for that use case. Please review / approve.

Reply:
I think this will still not work as expected with no middle name - the first line of the function creates an array based on params passed in and will *always* have a length of 3, but the 2nd element would be `undefined` if no middle names were passed in.
Probably easiest just to remove only the middle name element from the array...
```
} else if (options?.style === NameFormatStyle.firstLast) {
    names.splice(1, 1)
}
```